Source/ARTRealtimeChannel.m

```diff
@@ -1140,7 +1159,7 @@ - (BOOL)history:(ARTRealtimeHistoryQuery *)query callback:(ARTPaginatedMessagesC
     return [_restChannel history:query callback:callback error:errorPtr];
 }
 
-- (void)startDecodeFailureRecoveryWithChannelSerial:(NSString *)channelSerial error:(ARTErrorInfo *)error {
+- (void)startDecodeFailureRecoveryWithErrorInfo:(ARTErrorInfo *)error {
```

Comment:
Seeing as the signature has changed, why not 😉 ...

```suggestion
- (void)startDecodeFailureRecoveryWithErrorInfo:(ARTErrorInfo *const)error {
```

I could also suggest that `error` be renamed to `errorInfo`, given `error` in Objective-C codebases often refers to an `NSError` instance, however I recognise you've carried this over from the previous incarnation.